.github/workflows/e2e-tests.yaml

```diff
@@ -127,32 +126,6 @@ jobs:
           ACTION: "run_tests"
         run: |
           tests/e2e-kubernetes/scripts/run.sh
-      - name: Run Performance Tests
```

Comment:
Are these anything we care about?

Reply:
We were not using them to my knowledge. We can add them back properly in the future if we find value on it.